app/src/main/res/layout/activity_review_preview.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/album_art_size"
+        android:orientation="vertical"
+        android:background="@drawable/album_border">
+
+        <TextView
+            android:id="@+id/textview_album_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.5"
+            android:gravity="center"
+            android:textColor="@android:color/black"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            tools:text="Album Name"/>
+
+        <TextView
+            android:id="@+id/textview_artist_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="left"
+            android:layout_weight="0.5"
+            android:gravity="center"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            tools:text="Artist Name"/>
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/imageview_album_art_review"
+        android:layout_height="@dimen/album_art_size"
+        android:layout_width="match_parent"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_centerVertical="true"
```

Comment:
`layout_alignParentEnd`,  `layout_alignParentRight` and `layout_centerVertical` only work with `RelativeLayout` so will do nothing here